cli/smartling/api.go

```diff
@@ -166,17 +141,16 @@ var PutCommand = cli.Command{
 	},
 	Before: cmdBefore,
 	Action: func(c *cli.Context) {
-		if len(c.Args()) != 2 {
+		if len(c.Args()) != 1 {
 			log.Println("Wrong number of arguments")
-			log.Fatalln("Usage: put <local file> <remote file>")
+			log.Fatalln("Usage: put <local file>")
 		}
 
 		localpath := c.Args().Get(0)
-		remotepath := c.Args().Get(1)
```

Comment:
Hmm how are we getting away with not having a remote path? Seems weird to me not specifying the remote path, how do we know where the file is going?

Reply:
The API v2 does not allow you to specify a remote path: https://developer.smartling.com/v2.0/reference#post_projects-projectid-file

You just provide the fileUri (in this case the localepath) and you can then use that to request the file back.

Reply:
looks like two different fields to me?
```
file = The file contents to upload
fileUri = Value that uniquely identifies the uploaded file
```

Reply:
`file` is the actual _contents_ of the file, it does not having anything to do with the path of the file or identifying the file.  You can see how it is used here:  https://github.com/99designs/smartling/blob/master/api.go#L171

Reply:
Yeah exactly, the local file is just the content, so why remove the remotepath part? I'm not sure the localpath = remotepath is a good assumption. Some examples of other unix cli tools always use `<source> <target>` arguments

```
$ cp
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
```

```
$ scp
usage: scp [-346BCpqrv] [-c cipher] [-F ssh_config] [-i identity_file]
           [-l limit] [-o ssh_option] [-P port] [-S program]
           [[user@]host1:]file1 ... [[user@]host2:]file2
```

```
sftp put [-afPpr] local-path [remote-path]
```

Reply:
ahhh ok I see what you mean now, the penny has finally dropped! 

I'll patch that up.